src/config/normalize/index.ts

```diff
@@ -179,6 +179,9 @@ async function normalizedSimpleConfig(
         ...(commonOptions.mode || config.compilation?.mode
             ? { mode: commonOptions.mode || config.compilation?.mode }
             : {}),
+        ...(commonOptions.sourcemap ?? config.compilation?.sourcemap
```

Comment:
maybe receive false value

```suggestion
        ...(!isUndefined(commonOptions.sourcemap ?? config.compilation?.sourcemap)
```

Reply:
😳maybe `undefined` will also be passed in for farm use, I think.

Reply:
It's not important to us.